qutebrowser/browser/network/qutescheme.py

```diff
@@ -49,6 +49,15 @@ class QuteSchemeHandler(schemehandler.SchemeHandler):
 
     """Scheme handler for qute: URLs."""
 
+    handlers = dict()
+
+    @classmethod
+    def addHandler(cls, name):
```

Comment:
Please use `snake_case` naming - `camelCase` is used in qutebrowser for overrides of Qt methods only.